README.md

```diff
@@ -75,5 +75,13 @@ Current working directory for logging file paths.
 
 Logger function, called once at the end and once per file if `logFiles` is enabled.
 
+##### `options.includeZero: boolean = false`
+
+Set to true if you want to include results with no rows.
```

Comment:
verbiage could use some work. what does "include" mean here? 

i think the goal here is that it'll log something for an empty stream, so the wording should suggest that.

same for `onlyZero`